### ISSUE TITLE
mktemp incompatibility

### ISSUE THREAD

**Issue by surkamp** [WoeUSB/WoeUSB#49]
## Describe the bug ##

Slackware version of `mktemp` has no support to extra values before the `XXXXXX` tag.

## Fix suggestion ##

Remove the `.tempdir` part from `mktemp` arguments in `temp_directory` declaration.

## To Reproduce ##

1. Run the script on a Slackware 14.2 box
2. See error

## Teminal output ##

```
# ./woeusb-5.1.0.bash 
mktemp: cannot make temp dir /tmp/WoeUSB.XXXXXX.tempdir: Invalid argument
The command "temp_directory=$(
    mktemp         -d         -t         WoeUSB.XXXXXX.tempdir
)" failed with exit status "1", program is prematurely aborted
```

```
$ mktemp -V
mktemp version debianutils-2.7
```

## Environment ##

- OS: Slackware 14.2
- Bash: GNU bash, version 4.3.48(1)-release (x86_64-slackware-linux-gnu)
- WoeUSB: 5.1.0

**Comment by Lin-Buo-Ren** [comment]
Thanks for your report, I'll check it out.

**Comment by Lin-Buo-Ren** [comment]
```
vagrant@woeusb-slackware-14-2:~$ mktemp test.XXXXXX.test
mktemp: cannot create temp file test.XXXXXX.test: Invalid argument
vagrant@woeusb-slackware-14-2:~$ mktemp-gnu test.XXXXXX.test
test.IZszJ8.test
```

It appears that the GNU distribution of `mktemp` is also installed but in a different name.

Might just be simpler to simply change the template instead of supporting the installation.  :thinking:

**Comment by Lin-Buo-Ren** [comment]
Should be fixed in WoeUSB 5.1.2.
